contracts/TokenConverter/RiskFundConverter.sol

```diff
@@ -91,16 +98,23 @@ contract RiskFundConverter is AbstractTokenConverter {
             "ReserveHelpers: The pool doesn't support the asset"
         );
 
-        uint256 currentBalance = IERC20Upgradeable(asset).balanceOf(address(this));
+        IERC20Upgradeable token = IERC20Upgradeable(asset);
+        uint256 currentBalance = token.balanceOf(address(this));
         uint256 assetReserve = assetsReserves[asset];
         if (currentBalance > assetReserve) {
             uint256 balanceDifference;
             unchecked {
                 balanceDifference = currentBalance - assetReserve;
             }
-            assetsReserves[asset] += balanceDifference;
-            poolsAssetsReserves[comptroller][asset] += balanceDifference;
-            emit AssetsReservesUpdated(comptroller, asset, balanceDifference);
+            if (poolsAssetsDirectTransfer[comptroller][asset]) {
```

Comment:
We would need to add Core pool + BTC/ETH in the VIP: https://github.com/VenusProtocol/vips/pull/49

Plus, the USDT markets of IL

Reply:
Updated, https://github.com/VenusProtocol/vips/pull/49